content_sync/pipelines/concourse.py

```diff
@@ -249,3 +259,98 @@ def get_build_status(self, build_id: int):
     def abort_build(self, build_id: int):
         """Abort a build"""
         return self.api.abort_build(build_id)
+
+
+class ThemeAssetsPipeline(BaseThemeAssetsPipeline):
+    """
+    Concourse-CI pipeline for publishing theme assets
+    """
+
+    MANDATORY_SETTINGS = [
+        "CONCOURSE_URL",
+        "CONCOURSE_USERNAME",
+        "CONCOURSE_PASSWORD",
+        "GITHUB_WEBHOOK_BRANCH",
+        "SEARCH_API_URL",
+    ]
+
+    @staticmethod
+    def get_api():
+        """Get a Concourse API instance"""
+        return ConcourseApi(
+            settings.CONCOURSE_URL,
+            settings.CONCOURSE_USERNAME,
+            settings.CONCOURSE_PASSWORD,
+            settings.CONCOURSE_TEAM,
+        )
+
+    def __init__(self, api: Optional[ConcourseApi] = None):
+        """Initialize the pipeline API instance"""
+        super().__init__(api=api)
+        self.instance_vars = quote(
+            json.dumps({"branch": settings.GITHUB_WEBHOOK_BRANCH})
+        )
+
+    def _make_builds_url(self, job_name: str):
+        """Make URL for fetching builds information"""
+        return f"/api/v1/teams/{settings.CONCOURSE_TEAM}/pipelines/ocw-theme-assets/jobs/{job_name}/builds?vars={self.instance_vars}"
+
+    def _make_pipeline_config_url(self):
+        """Make URL for fetching pipeline info"""
+        return f"/api/v1/teams/{settings.CONCOURSE_TEAM}/pipelines/ocw-theme-assets/config?vars={self.instance_vars}"
+
+    def _make_job_url(self, job_name: str):
+        """Make URL for fetching job info"""
+        return f"/api/v1/teams/{settings.CONCOURSE_TEAM}/pipelines/ocw-theme-assets/jobs/{job_name}?vars={self.instance_vars}"
+
+    def _make_pipeline_unpause_url(self):
+        """Make URL for unpausing a pipeline"""
+        return f"/api/v1/teams/{settings.CONCOURSE_TEAM}/pipelines/ocw-theme-assets/unpause?vars={self.instance_vars}"
+
```

Comment:
May want to consider moving these to the base pipeline to avoid nearly identical code, see this PR:  https://github.com/mitodl/ocw-studio/pull/950/files#diff-979f137beba738446b3fc46c89be033a4f04e2e285ce7e68d3028b0e1b83c11fR104-R138

In this case, `ocw-theme-assets` would be a value for the `version` parameter (which I will probably change to `pipeline_name` in my PR).
For example:
```
    def _make_pipeline_config_url(self, pipeline_name: str):
        """Make URL for fetching pipeline info"""
        return f"/api/v1/teams/{settings.CONCOURSE_TEAM}/pipelines/{pipeline_name}/config?vars={self.instance_vars}"
```

and call it for `ThemeAssetsPipeline` in `upsert_theme_assets_pipeline`. like this:
```
url_path = self._make_pipeline_config_url(self.pipeline_name)
```

where `self.pipeline_name` could be set in the `init()` function to `"ocw-theme-assets"`